OJP/OJP_Common.xsd

```diff
@@ -188,6 +188,28 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
+	<xs:complexType name="VehicleFilterStructure">
+		<xs:annotation>
+			<xs:documentation>Filter for Vehicles</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="Exclude" type="xs:boolean" default="true" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Whether Vehicle and TRAIN NUMBERS in list are to include or exclude from search. Default is exclude.</xs:documentation>
```

Comment:
Whether to include or exclude given VehicleRefs and TRAIN NUMBERS in the list from the search. Default is exclude.

Reply:
copied your text